src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -509,7 +507,7 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
                 loading={isCommittingCreateOfferOrderMutation}
                 onClick={this.handleCreateOfferOrder.bind(this)}
               >
-                Make offer
+                Make an Offer
```

Comment:
Should this be capitalized like this? 🤔 
I'm seeing on the ticket that it's marked that way, but might be worth confirming with PM and design; I think we moved away from that way of capitalizing.

Reply:
Same with "Price on request"